nautobot/extras/tests/test_filters.py

```diff
@@ -1084,7 +1084,7 @@ def setUpTestData(cls):
         ).validated_save()
 
     def test_type(self):
-        params = {"type": "one-to-many"}
+        params = {"type": ["one-to-many"]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
```

Comment:
With all of these string to list changes I feel like this is going to be a breaking change, if not at least for all apps who's tests were expecting to filter on a choice of one.

Reply:
Yeah, it's only a breaking change to things that are programmatically using the filterset (UI and API filtering should "just work") but that's definitely worth calling out.

Reply:
A bug fix that is a breaking change! What is the world coming to!

Reply:
I guess illd have to rebase to next?

Reply:
Since this isn't a ground-breaking bug (I'm not seeing hard-stop issues for users) can I request this go into `next` so as to not induce unnecessary churn for apps and their automated tests?

Reply:
I'd be fine with keeping this in `develop` as long as we add a clear note in the release-notes.